src/testutils.ts

```diff
@@ -137,7 +137,7 @@ export function createExample<T>(type: Type<T>, seed?: number): T {
 class CreateExampleVisitor implements Visitor<unknown> {
     constructor(private seed = 1) {}
 
-    visitArrayType(type: ArrayType<BaseTypeImpl<unknown, unknown>, unknown, unknown[]>): unknown {
+    visitArrayType(type: ArrayType<BaseTypeImpl<unknown>, unknown, unknown[]>): unknown {
```

Comment:
typescript-eslint complained that `unknown` is the default, so should not have been mentioned.